web-workout-tracker/workout/src/index.js

```diff
@@ -1,10 +1,9 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
-import './index.css';
 import App from './App';
 import { BrowserRouter as Router, Route } from "react-router-dom";
-import * as serviceWorker from './serviceWorker';
 
+import GlobalStyle from './design/global_style'
```

Comment:
I am good with these additions but i am not sure why all of the checks with netlify are failing